packages/trpc/server/routers/loggedInViewer/setDestinationCalendar.handler.ts

```diff
@@ -61,32 +61,22 @@ export const setDestinationCalendarHandler = async ({ ctx, input }: SetDestinati
     where = { eventTypeId };
   } else where = { userId: user.id };
 
-  await prisma.destinationCalendar.upsert({
+  await DestinationCalendarRepository.upsert({
     where,
     update: {
       integration,
       externalId,
-      ...(!isDomainWideDelegationCredential({ credentialId })
-        ? {
-            credentialId,
-          }
-        : {
-            domainWideDelegationCredentialId,
-          }),
```

Comment:
@hariombalhara is it okay to remove this?

credentialId could be -1 for domain wide credential

Reply:
yeah confirmed this doesn't work

![Screenshot 2024-12-03 at 4 07 17 PM](https://github.com/user-attachments/assets/a739c6ed-3ae0-4371-b4dd-48024b8728eb)